src/LibYear/Settings.cs

```diff
@@ -28,4 +28,8 @@ public class Settings : CommandSettings
 	[CommandOption("-a|--limit-any")]
 	[Description("fails if any dependency is more libyears behind than this value")]
 	public double? LimitAny { get; set; }
+
+	[CommandOption("-r|--recursive")]
+	[Description("in all workding directory sub-folders")]
```

Comment:
Let's make this a little clearer, something like:
> search recursively for all compatible files, even if one is found in a directory passed as an argument

And we'll need to update the `README` file, too!